src/rt/minfo.d

```diff
@@ -235,12 +231,82 @@ struct ModuleGroup
             memset(flags, 0, flagbytes);
         }
 
+
+        // build the edges between each module. We may need this for printing,
+        // and also allows avoiding keeping a hash around for module lookups.
+        int[][] edges = (cast(int[]*)malloc((int[]).sizeof * _modules.length))[0 .. _modules.length];
```

Comment:
Is that really such a good idea?
Space for all edges could be quite big, right?
And we almost never need to print cycles.